iothub/device/src/Edge/InstalledCertificateValidator.cs

```diff
@@ -33,9 +33,9 @@ private void SetupCertificateValidation()
         {
             Debug.WriteLine("InstalledCertificateValidator.SetupCertificateValidation()");
 
-            foreach (var cert in this._certs)
```

Comment:
we don't need to create a new store object on every single cert

Reply:
I don't see anyone referencing this private variable `store`, is it even required? or is the only validation that the certificate can be added to **_a_** store?